modules/util/portal-tools-rest-builder/bnd.bnd

```diff
@@ -33,7 +33,5 @@ Main-Class: com.liferay.portal.tools.rest.builder.RESTBuilder
 	@com.liferay.portal.kernel-*.jar!/com/liferay/portal/kernel/util/*,\
 	@com.liferay.portal.tools.java.parser-*.jar!/com/liferay/portal/tools/java/parser/*.class,\
 	@com.liferay.portal.tools.java.parser-*.jar!/com/liferay/portal/tools/java/parser/util/JavaParserUtil.class,\
-	@commons-io-[0-9.]*.jar!/org/apache/commons/io/FileUtils.class,\
```

Comment:
@jeffwu0724 This removal is kind of risky, as this tool is going to be used for all released versions, please pay attention to this change once this pull is merged. If it breaks anything, try bringing this back.

Tina.